src/main/java/ru/fallindawn/booking/registration/service/RegistrationService.java

```diff
@@ -0,0 +1,10 @@
+package ru.fallindawn.booking.registration.service;
+
+import org.springframework.stereotype.Service;
+import ru.fallindawn.booking.registration.dto.RegistrationDto;
+
+@Service
```

Comment:
think we no need it here, because @ Service placed in RegistrationServiceImplementation